modules/users/server/controllers/users/users.authentication.server.controller.js

```diff
@@ -79,10 +79,29 @@ exports.signout = function(req, res) {
 };
 
 /**
+ * OAuth provider call
+ */
+exports.oauthCall = function(strategy, scope) {
+	return function(req, res, next) {
+		// Set redirection path on session.
+		// Do not redirect to a signin or signup page
+		if (['/authentication/signin', '/authentication/signup'].indexOf(req.query.redirect_to) === -1) {
```

Comment:
Instantiating this array every time will be costly, let's store a constant reference somewhere, no?